### DISCUSSION TITLE
Passcode Unlock

### DISCUSSION THREAD

**Discussion by Dzoge** [keepassxreboot/keepassxc#6861] [Ideas]
I have been using keepassxc for a while now and I love this tool.

It would be an awesome feature to be able to set a simple passcode, so that once I unlock the database the app would remember the password in the memory and let me unlock the database quicker while the app is running.

Now I have to enter my very long password quite a few times a day and it it a bit frustrating.

Would you consider adding this kind of feature?

Cheers <3

**Comment by droidmonkey** [comment]
Yes it's called quick unlock and we are looking to implement it.

**Reply by dclong** [reply]
And can make keepassxc-cli support fingerprint too?
